src/util/wallet-utils/src/getAddress.ts

```diff
@@ -99,10 +106,15 @@ export const getAddressesFordefi: getAddresses = async (params) => {
 };
 
 export const getAddressesAsigna: getAddresses = async (params) => {
+  return Promise.race([getBTCAddressAsigna(params), getStxAddressAsigna()]);
```

Comment:
qq: what happens calling `params.action()` (ie, `connect` from `useAsignaConnect` from `@asigna/btc-connect`) when using stx-asigna?

Reply:
it's ignored since getStxAddress asigna wins the race

Reply:
Yeah, but assuming we are not racing, what happens to that call? It stalls forever, return an error or wreak havoc?

Reply:
Good q it will be basically waiting in the bg but nothing to worry about here memory leak wise according to [this guy](https://stackoverflow.com/posts/61136891/revisions) at least lol

Reply:
Got it, thanks. We should double check what happens now that we are using `any` (I'm thinking about a scenario where the btc asigna stalls while the stx asigna fails), but I guess it's fine for now.